tests/ui/pages/desktop/devhub.py

```diff
@@ -53,10 +54,13 @@ def login(self, email, password):
         self.wait.until(
             lambda _: self.is_element_displayed(*self._avatar_locator)
         )
+        return self
 
     @property
     def logged_in(self):
-        return self.is_element_displayed(*self.header._sign_in_locator)
+        if self.is_element_displayed(*self.header._sign_in_locator):
+            return False
+        return True
```

Comment:
What changed? Doesn't the previous `return` do exactly the same?

Reply:
The logic goes: If the sign in locator is displayed, you are not ```logged_in```. If it is not displayed then you are. Just using ```is_displayed```  didn't correlate logically to how the property would be used.

Reply:
Ah, you could du `return not self.is_element_displayed(*self.header._sign_in_locator)` then I'd guess?

Reply:
Ah, yeah, being all pythonic and stuff.